goa/Classes/menu/MainMenuHome.cpp

```diff
@@ -78,6 +78,9 @@ bool MainMenuHome::init()
     //    }
 
     cocos2d::ui::Button *backButton = createBackButton();
+    Size frameSize = Director::getInstance()->getOpenGLView()->getFrameSize();
```

Comment:
Can you create a public static variable for _frameSize_ and _aspectRatio_?

Reply:
I had tried that making the variables global but some of the files are linked producing a redeclaration error during the compilation.